torch/utils/hipify/hipify_python.py

```diff
@@ -702,9 +708,13 @@ def pattern(self):
         meta_data = value[1:]
         if constants.API_CAFFE2 not in meta_data:
             PYTORCH_TRIE.add(src)
+            # if src is already in PYTORCH_MAP and dst belongs to API_SOLVER
+            # do not overwrite PYTORCH_MAP, store dst separately
+            if constants.API_SOLVER in meta_data and PYTORCH_MAP.get(src, ""):
+                PYTORCH_SOLVER_MAP[src] = dst
             # if src is already in PYTORCH_MAP and dst belongs to API_SPARSE
             # do not overwrite PYTORCH_MAP, store dst separately
-            if constants.API_SPARSE in meta_data and PYTORCH_MAP.get(src, ""):
+            elif constants.API_SPARSE in meta_data and PYTORCH_MAP.get(src, ""):
```

Comment:
Just unsure if we should use `if` vs. `elif` here, in case both `SOLVER` and `SPARSE` got the same stuff.
Or a comment would address this temporarily.